build.sbt

```diff
@@ -27,6 +27,19 @@ scalacOptions ++= Seq(
   "-Ywarn-unused:imports",
 )
 
+scmInfo := Some(
```

Comment:
I completely misunderstood the intent of this PR. 

I thought it was about making sure that a project that uses sbt-daffodil could have its LICENSE/NOTICES etc. included in its jars, and appropriate metadata into poms. 

But this is just about this sbt plugin project itself.

Reply:
Correct, this is just about making Apache happy when we eventually create a release of this plugin. I'll modify the commit message to make that more obvious before pushing.

Note that we do have issue https://github.com/apache/daffodil-sbt/issues/7 about copying REAMDEs into schema project jars. I'll update that to also mentions copying in LICENSE information.